src/frontend/Sider/Behandling/Inngangsvilkår/Vilkårperioder/EndreVilkårperiodeRad.tsx

```diff
@@ -13,10 +14,10 @@ import { EndreAktivitetForm } from '../Aktivitet/EndreAktivitetRad';
 import { EndreMålgruppeForm } from '../Målgruppe/EndreMålgruppeRad';
 import { KildeVilkårsperiode, VilkårPeriode, VilkårPeriodeResultat } from '../typer/vilkårperiode';
 
-const TabellRad = styled(Table.Row)`
+const TabellRad = styled(Table.Row)<{ $feilmeldingVises: boolean }>`
     .navds-table__data-cell {
         border-color: transparent;
-        vertical-align: top;
+        vertical-align: ${(props) => (props.$feilmeldingVises ? 'top' : 'center')};
```

Comment:
Mulig du får forklare denne på nytt for meg i morgen 😆

Reply:
Gir det et forsøk her. Hovedproblemet er at man ønsker at ikonene (resultat og kilde) skal være sentrert på raden, men når det kommer feilmelding så ser det rart ut om man kun sentrerer. 

Hvordan det ser ut i ikke-redigering/visningsmodus (ting er sentrert vertikalt): 
<img width="651" alt="image" src="https://github.com/navikt/tilleggsstonader-sak-frontend/assets/46678893/87865597-fbc4-464a-9d51-f0dad55ff850">

Om man alltid bruker `vertical-align: center` og det kommer en feilmelding:
<img width="844" alt="image" src="https://github.com/navikt/tilleggsstonader-sak-frontend/assets/46678893/7969441b-b180-4afd-9fee-f97611248150">

Derfor brukes center med mindre feilmelding vises, da brukes "top". Dette gjør at det er et lite "skift" men mindre enn at alt sentreres:
_Uten feilmelding:_ 
<img width="767" alt="image" src="https://github.com/navikt/tilleggsstonader-sak-frontend/assets/46678893/5f41e0d7-62f7-4f06-b4ef-2428960140a4">

_Med feilmelding:_ 
<img width="841" alt="image" src="https://github.com/navikt/tilleggsstonader-sak-frontend/assets/46678893/6fa3b2ae-b06e-4971-b355-d68f6270e34b">